doc/classes/StyleBox.xml

```diff
@@ -70,12 +71,60 @@
 	</methods>
 	<members>
 		<member name="content_margin_bottom" type="float" setter="set_default_margin" getter="get_default_margin">
+			The bottom margin for the "contents" of this style box.
+
```

Comment:
Each line is a paragraph, so empty lines means adding empty paragraphs. Line breaks below will also mean that each line will be a separate paragraph.

Reply:
Fixed it myself.